bin/kano-uixinit

```diff
@@ -18,6 +18,15 @@
 
 . /usr/share/kano-toolset/logging.sh
 
+# If we are in Kano v3, jump to Dashboard Mode, unless instructed otherwise
+if [ "${DESKTOP_MODE}" != "1" ]; then
```

Comment:
@skarbat is not a problem to have this line above everything? Shouldn't it be done after [this line](https://github.com/KanoComputing/kano-desktop/blob/dashboard-mode/bin/kano-uixinit#L83) at least?